### ISSUE TITLE
Battle for Undercity

### ISSUE THREAD

**Issue by ninjahobbit** [dalaranwow/dalaran-wow#5414]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: On my Lock (Balash) "Battle for Undercity" quest bugged out.  I abandoned and tried to re-take but can't.  Now whenever I go to Orgrimmar there are no NPCs and it is as if I was on the quest still.

**Expected behaviour**: I should be able to go into Orgrimmar and interact as normal

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![image](https://user-images.githubusercontent.com/38582138/39082504-5fc584c8-4554-11e8-9ef4-18dd12a85a32.png)

**Comment by ninjahobbit** [comment]
PS Cache cleared, WTF deleted and no improvement
